variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "policy_json" {
+  description = "Policy to attach to the user"
+  type        = string
+}
+
+variable "policy_name" {
+  description = "Name to give to the policy attached to the user"
+  type        = string
+}
+
+variable "secret_name" {
+  description = "AWS Secrets Manager secret name. To store the AWS IAM user's access keys"
+  type        = string
+}
```

Comment:
@rbapst-tamedia I'm a fan of sensible defaults, with easy override option. Are these two variables required from a requirement perspective?

IMHO most of the time the user of the module doesn't care about the `policy_name` or the `secret_name`, so it would be nice to have defaults (or `locals`) with the option to override them through vars. What do you think? :)

Another comment on the `secret_name` input: it's not totally clear what I, as a user of the module, should do with an input called `secret_name`. What is it for? Reading the description makes it clear, but maybe there's a more precise name for it :)

Reply:
policy_name removed, name is only displayed in the user policy and terraform has this as an option.

for secret_name we could also have it optional and default to something which matches our naming convention for secrets, but I hesitate because it's a public module.